api/v1beta1/cloudstackmachine_types.go

```diff
@@ -52,6 +52,10 @@ type CloudStackMachineSpec struct {
 	// +optional
 	DiskOffering CloudStackResourceDiskOffering `json:"diskOffering,omitempty"`
 
+	// CloudStack ISO attachment, ISO is attached before VM starts.
+	// +optional
+	ISOAttachment CloudStackResourceIdentifier `json:"ISOAttachment,omitempty"`
```

Comment:
I'm concerned about the standardization of the json tags. Over here it matches the struct member,s name but for sshkey it does not. Can this be standardized ?